graylog2-server/src/main/java/org/graylog/scheduler/JobDefinitionDto.java

```diff
@@ -64,6 +58,7 @@ public static Builder create() {
         @Id
         @ObjectId
         @JsonProperty(FIELD_ID)
+        @Nullable
         public abstract Builder id(String id);
```

Comment:
The `id` parameter should be `Nullable`, not the return value of the `#id()` builder method.

```suggestion
        public abstract Builder id(@Nullable String id);
```

Reply:
Thanks for spotting this. The annotation was a leftover from an experiment and not intended to be kept. I removed it again.